CHANGES.rst

```diff
@@ -21,6 +21,13 @@ Unreleased
 - The ``after`` argument for :meth:`.conversations` will now have to be included in
   ``params`` keyword argument.
 
+7.6.0 (2021/12/23)
+------------------
+**Added**
+
+- Add method :meth:`~.Collection.update_display_layout` to :class:`.Collection` class to support
```

Comment:
This needs to go in the unreleased section.

Reply:
Oh, shoot. Do you know when you'll update PRAW? I was hoping to use this new feature after this gets merged to master.